### PR TITLE
Display logout button in console when properly logged in

### PR DESCRIPTION
**Pull request by Matthbo**
The button in the frontend does display now when a username is gotten from the backend.

PR #7101 should've made the `/logout` endpoint work but somehow it won't work in Firefox and is iffy in Chrome, this should be looked into.

### CODE REVIEW

File: console/frontend/src/main/frontend/src/app/views/login/login.component.html [hunk 1]
```diff
@@ -27,6 +27,7 @@ <h3>Frank<strong class="text-frank">!</strong>Console</h3>
           placeholder="Username"
           required
           autofocus
+          disabled
```
Comment (nielsm5) [inline console/frontend/src/main/frontend/src/app/views/login/login.component.html:30]: ?

Comment (Matthbo) [reply console/frontend/src/main/frontend/src/app/views/login/login.component.html:30]: The form is currently not being used so I disabled the inputs. This way you only have to submit to go to the main view of the application

File: console/frontend/src/main/frontend/src/app/views/login/login.component.ts [hunk 2]
```diff
@@ -24,19 +25,26 @@ export class LoginComponent implements OnInit {
...
+    private router: Router,
   ) {}
 
   ngOnInit(): void {
-    this.authService.loggedin(); // Check whether the client is logged in.
+    if (this.authService.isLoggedIn()) {
+      this.router.navigate(['/']);
+    }
 
     window.setTimeout(() => {
```
Comment (philipsens) [inline console/frontend/src/main/frontend/src/app/views/login/login.component.ts:36]: Couldn't this timeout be substituted with ngAfterViewInit?

Comment (Matthbo) [reply console/frontend/src/main/frontend/src/app/views/login/login.component.ts:36]: I guess, I'm not fully sure on what the intent was originally
I'll update it

**Review by philipsens — APPROVED** [review]

**Pull request merged**